src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -135,6 +138,41 @@ public void savePatientVitals() {
 
 	@Then("the system adds patient vitals into the vitals table")
 	public void systemAddsPatientVitals() {
-		assertTrue(patientCaptureVitalsPage.containsText("Vitals"));
+		assertNotNull(visitsDashboardPage.getEncountersCount());
+	}
+	
+	@When("a user clicks on edit vitals icon from patient visits dashboard")
+	public void loadEditVitalsPage() {
+		editVitalsPage = (EditVitalsPage) visitsDashboardPage.goToEditVitalsPage().waitForPage();
+	}
+	
+	@And("the system loads the edit vitals page")
+	public void systemloadsEditPatientVitalsPage() {
+		assertTrue(textExists("Edit: Vitals"));
```

Comment:
we can rather assert that the EditVitals page is loaded successfully than asserting the text on the page;
```js
@And("the system loads the edit vitals page")
	public void systemloadsEditPatientVitalsPage() {
	     assertPage(editVitalsPage.waitForPage());}
```